R/constants.R

```diff
@@ -20,6 +20,9 @@ FLAG_PROTECT_FLOAT <- "FLAG_PROTECT_FLOAT"
 HISTORICAL_YEARS <- 1971:2010
 FUTURE_YEARS <- seq(2015, 2100, 5)
 IMF_GDP_YEARS <- 2010:2020
+BASE_YEARS <- c(1975, 1990, 2005, 2010)
```

Comment:
Is this `BASE_YEARS` a d.s.-wide setting, or water-specific? Should it be qualified, e.g. `water.BASE_YEARS`? @mihejazi @pkyle